### ISSUE TITLE
Load the banner before triggering if animated (source pre-caching)

### ISSUE THREAD

**Issue by rkluvanec** [hackal/Exp#63]
As of now, the banner is only injected and loaded after being triggered when animated. This causes browsers with a slower connection to display a banner that is not fully loaded and load it afterward which looks bad.
Could we figure out a way of pre-loading all the dependencies for the banner so that it is ready to be triggered to the website?

**Comment by hackal** [comment]
Yes, that is possible :) simple trick that would scan the HTML / CSS, look at the media files and do a prefetch call either through CSS or JS. Sounds like a reasonable feature to optimize performance.
